common/src/hash.rs

```diff
@@ -14,15 +14,18 @@ where
     output
 }
 
+/// A 32-byte hash prefix.
```

Comment:
The word prefix here suggests to me that this is only the beginning of a 32-byte hash (like the `0x` to indicate it is a hex representation). Is this intended?

Reply:
Yes, it is the first 4 bytes in a 32-byte keccak hash. I'll clarify in the comments.